src/util/root.js

```diff
@@ -0,0 +1,3 @@
+export default (typeof self === 'object' && self.self === self && self) ||
```

Comment:
No, but this was an experiment because Rollup was erroring on it. It also errored on iDOM so I just nuked it and used only Webpack 2. We can restore the dependency now, if we want.